### PR TITLE
fix: changelog history

### PR DESCRIPTION
**Pull request by simbleau**
fixes the changelog history requested to be a separate PR by @waywardmonkeys

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,58 +30,69 @@ This release has an [MSRV][] of 1.75.
...
+- Added `vello_svg::render_tree`, which takes a usvg::Tree and renders to a provided vello scene with and error handler. ([#18] by [@simbleau])
 
 ### Changed
 
-- Updated to vello 0.2
-- Updated to usvg 0.42
-- Renamed `render_tree` to `append_tree`
-- Renamed `render_tree_with` to `append_tree_with` and removed the `Result<(), E>` return type for the error handler.
+- Updated to vello 0.2 ([#18] by [@simbleau])
+- Updated to usvg 0.42 ([#18] by [@simbleau])
```
Comment (DJMcNab) [inline CHANGELOG.md:42]:
```suggestion
- Updated to Vello 0.2.0 and `usvg` 0.42 ([#18] by [@simbleau])
```
or 
```suggestion
- Updated to `vello` 0.2.0 and `usvg` 0.42 ([#18] by [@simbleau])
```

Personally, I prefer the former, but I think the latter more strictly follows our current scheme.
I suspect that the "referring to the crates.io name" is not a case that we should have.

**Review by waywardmonkeys — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,58 +30,69 @@ This release has an [MSRV][] of 1.75.
...
 
-- Initial release
+- Initial release. ([#1] by [@simbleau])
 
 [@nicoburns]: https://github.com/nicoburns
 [@waywardmonkeys]: https://github.com/waywardmonkeys
+[@simbleau]: https://github.com/simbleau
+[@nixon-voxell]: https://github.com/nixon-voxell
+[@MarijnS95]: https://github.com/MarijnS95
+[@DasLixou]: https://github.com/DasLixou
```
Comment (waywardmonkeys) [inline CHANGELOG.md:85]: Although if you want to alphabetize this list, that's fine / good.

**Review by DJMcNab — APPROVED** [review]
~~Good to get this synced up. Just one detail on our capitalisation. Could you also change the mention of Vello added in #28?~~

The capitalisation question doesn't need to be resolved in this PR

File: CHANGELOG.md (hunk 1 above)
Comment (DJMcNab) [reply CHANGELOG.md:42]: Discussion in https://xi.zulipchat.com/#narrow/stream/419691-linebender/topic/Linebender.20capitalization.20scheme

**Pull request merged**